extra/exports_xxhash

```diff
@@ -0,0 +1,12 @@
+# Symbols to be exported from xxHash library
+
+tnt_XXH32
+tnt_XXH32_copyState
+tnt_XXH32_digest
+tnt_XXH32_reset
+tnt_XXH32_update
+tnt_XXH64
+tnt_XXH64_copyState
+tnt_XXH64_digest
+tnt_XXH64_reset
+tnt_XXH64_update
```

Comment:
The fact that you added 2 separate files for curl and xxhash raises a question why didn't you add them for iconv,  (tnt_iconv_* functions), for csv, for crypto? Maybe have a single exports file for everything? Otherwise I don't understand when a new file is needed and when the main exports must be used.

Curl I can understand because it seems to be exported conditionally. But xxhash is exported together with the main exports file always from what I see.

Reply:
Moved to exports.